charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.47.0
-version: 24.4.0
+version: 24.4.1
```

This change should be considered minor:

```suggestion
version: 24.5.0
```

---

Agree, it should be a minor version.